attributes/network/9_account_features/cddl/1_account_multisig.cddl

```diff
@@ -13,35 +13,35 @@ multisig-transaction-state /= 0  ; Pending
                             / 3  ; Withdrawn
                             / 4  ; Expired
 
-; List of transaction types added by this feature.
-; These should be listed in ledger.list.
-ledger-transaction-info /= account-multisig-transaction-submit
-                         / account-multisig-transaction-approve
-                         / account-multisig-transaction-revoke
-                         / account-multisig-transaction-execute
-                         / account-multisig-transaction-withdraw
-                         / account-multisig-transaction-set-defaults
-                         / account-multisig-transaction-expire
-
-; Set of indices for multisig transaction entries.
-account-transaction-type-multisig-submit        = [9, [1, 0]]
-account-transaction-type-multisig-approve       = [9, [1, 1]]
-account-transaction-type-multisig-revoke        = [9, [1, 2]]
-account-transaction-type-multisig-execute       = [9, [1, 3]]
-account-transaction-type-multisig-withdraw      = [9, [1, 4]]
-account-transaction-type-multisig-set-defaults  = [9, [1, 5]]
-account-transaction-type-multisig-expire        = [9, [1, 6]]
-
-ledger-transaction-type /= account-transaction-type-multisig-submit
-                         / account-transaction-type-multisig-approve
-                         / account-transaction-type-multisig-revoke
-                         / account-transaction-type-multisig-execute
-                         / account-transaction-type-multisig-withdraw
-                         / account-transaction-type-multisig-set-defaults
-                         / account-transaction-type-multisig-expire
-
-account-multisig-transaction-submit = {
-    0 => account-transaction-type-multisig-submit,
+; List of event log types added by this feature.
+; These should be listed in events.list.
+event-info /= account-multisig-event-info-submit
+           /  account-multisig-event-info-approve
+           /  account-multisig-event-info-revoke
+           /  account-multisig-event-info-execute
+           /  account-multisig-event-info-withdraw
+           /  account-multisig-event-info-set-defaults
+           /  account-multisig-event-info-expire
+
+; Set of indices for multisig event entries.
+account-event-type-multisig-submit        = [9, 1, 0]
```

Comment:
Why did you change the format here?

Ref. spec:
```
; An attribute can have one or many attribute arguments.
attribute-arg = ()
attribute-id = uint
attribute = attribute-id / [attribute-id, * attribute-arg]
```

Reply:
Yeah I was going back and forth here, I guess I forgot to change it back.